### PR TITLE
SageMaker Integration Work

### PR DESCRIPTION
**Pull request by chrisking**
This contains the following:

* A markdown file describing how to get started with SageMaker and Forecast
* A JSON copy of the IAM Policy
* Updated script for applying policy with IAM changes
* An updated notebook optimized for SageMaker instances

### CODE REVIEW

File: SageMaker_Tutorial.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Next, ensure you are in the US-West-2 Region, do do that look in the top left corner, if it says `Oregon` next to support
+that is correct, otherwise select `Oregon` from the drop-down.
+
+Under `Find services` in the main body of the page, enter `SageMaker`, then select it from the drop-down.
+
+To the left, will see a category titled `Notebook` inside that, click `Notebook instances`.
+
+Click the orange `Create notebook instance` button.
+
+Give the instance a name unique in the account you are using. If a shared account, place your name first like `ChrisKingForecastDemo`. The default Instance
```
Comment (akhilrazdan) [inline SageMaker_Tutorial.md]: Change `ChrisKingForecastDemo` to something generic, so it's more AWS like.

Comment (akhilrazdan) [inline SageMaker_Tutorial.md]: Add a more generic name for `ChrisKingForecastDemo`

File: SageMaker_Tutorial.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+The Next component to change is the IAM role. Under the drop-down click `Create a new role`. Then for S3, select `Any S3 Bucket`, finally click `Create role`.
+Note that the role itself has become a link. Open that link in a new tab.
+
+Here you will update the policies of your instance to allow it to work with Forecast. Click the `Attach policies` button.
+
+Next click the `Create policy` button at the top. In the new page, click the `JSON` tab.
+
+Erase all of the content that is in the editor and paste the content in [IAM_Policy.json](IAM_Policy.json).
+
+After pasting, click the `Review policy` button. Give the policy again a personalize name like `ChrisKingForecastIAMPolicy`.
```
Comment (akhilrazdan) [inline SageMaker_Tutorial.md]: Generic Name

File: SageMaker_Tutorial.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+To begin, click `Open Jupyter`, this will take you to the default interface for the Notebook Instance.
+
+Click `New` then click `Terminal`, this will open a BASH shell for you on this instance. 
+
+Enter the following commands to clone this repository onto this instance:
+
+```
+cd SageMaker
+git clone https://github.com/aws-samples/amazon-forecast-samples.git
```
Comment (akhilrazdan) [inline SageMaker_Tutorial.md]: You verified the git setup doesn't require any additional stuff on Sagemaker, right?

File: notebooks/Getting_started_with_Forecast-SageMaker.ipynb [hunk 4]
```diff
@@ -0,0 +1,757 @@
+{
```
Comment (akhilrazdan) [inline notebooks/Getting_started_with_Forecast-SageMaker.ipynb]: This file is almost the same as the default file, with minor documentation updates and the clean up part. Is there anything big i am missing? In that case, you can just add it to the main notebook?

File: SageMaker_Tutorial.md (hunk 3 above)
Comment (chrisking) [reply SageMaker_Tutorial.md]: Yes

**Comment by chrisking** [comment]
@akhilrazdan I believe all of the changes you requested have been made, please let me know if there's anything else.

**Pull request merged**